proposals/NNNN-ServiceStatusUpdateToHMI.md

```diff
@@ -0,0 +1,114 @@
+# Service Status Update to HMI
+
+* Proposal: [SDL-NNNN](NNNN-ServiceUpdateToHMI.md)
+* Author: [Robin Kurian](https://github.com/robinmk)
+* Status: **Awaiting Review**
+* Impacted Platforms: [Core / RPC]
+
+## Introduction
+
+This proposal is about adding a new RPC called 'OnServiceUpdate' which will be used by the HMI layer to know the status of certain services from SDL Core.
+
+
+## Motivation
+
+Today, when a Mobile Navigation application is activated there is a series of steps executed begining with the app sending the StartService request for the Video Service and ending with the module acknowledging this request in a successful scenario. These steps include getting the current system time, performing a policy table update, decrypting certificates and ensuring validity of the certificates. Any of these steps could fail and since there is no mechanism for updating HMI with the status of the service, the HMI has no means of providing accurate feedback to the user on the status of the system or what steps to take in case of an error.
```

Comment:
Please change to: 

Please note the spelling change of **beginning**

When a Mobile Navigation application is activated there is a series of steps taken; **beginning** with the app sending the StartService request for the Video Service and ending with the module acknowledging this request in a successful scenario. These steps include getting the current system time, performing a policy table update, decrypting certificates and ensuring validity of the certificates. 
Any of these steps could fail and since there is no mechanism for updating HMI with the status of the service, the HMI has no means of providing accurate feedback to the user on the status of the system or what steps to take in case of an error.